src/Collectors/JobCollector.php

```diff
@@ -105,6 +106,17 @@ protected function stopTransaction(string $transaction_name, int $result): void
         $this->agent->collectEvents($transaction_name);
     }
 
+    protected function addMetadata(string $transaction_name, Job $job): void
+    {
+        $this->agent->getTransaction($transaction_name)->setCustomContext([
```

Comment:
ok, so this is a setter, but [it actually merges existing context with this](https://github.com/philkra/elastic-apm-php-agent/blob/773ef84e5f524d0f7dc0950e9f434255258e09f5/src/Events/EventBean.php#L215) (which I think is good in this case).